heron_wsgi/admin_lib/heron_policy.py

```diff
@@ -871,10 +871,10 @@ def _script():
         cwd = Path('.', open=io_open, joinpath=joinpath, listdir=listdir)
         logging.basicConfig(level=logging.DEBUG, stream=stderr)
 
-        sys_path.append('..')
+        sys_path.append('./heron_wsgi/')
         import traincheck
 
-        ini = cwd / 'integration-test.ini'
+        ini = cwd / 'production.ini'
```

Comment:
again, I use `integration_test.ini` regularly. Can you add a check to see if an ini file is given in `argv`?

Reply:
i can't add an argument so long as `argv[2]` is being checked the way it is.  thats going to have to change to `--attrs` or something.  or i can hardcode argv[2] to always be the path of the config file and move attrs back.  either way something is going to have to change